includes/DataObjects/RequestData.php

```diff
@@ -23,9 +23,13 @@ class RequestData extends DataObject
 {
     const TYPE_IPV4 = 'ipv4';
     const TYPE_IPV6 = 'ipv6';
+    const TYPE_CONFIRM_IPV4 = 'confirm-ipv4';
+    const TYPE_CONFIRM_IPV6 = 'confirm-ipv6';
     const TYPE_EMAIL = 'email';
     const TYPE_USERAGENT = 'useragent';
+    const TYPE_CONFIRM_USERAGENT = 'confirm-useragent';
     const TYPE_CLIENTHINT = 'clienthint';
+    const TYPE_CONFIRM_CLIENTHINT = 'confirm-clienthint';
```

Comment:
Nit: Not really material to this PR, mostly just flagging so you're aware - `TYPE_IPV4`, `TYPE_IPV6`, and `TYPE_EMAIL` all appear to be unused.